src/components/DisplayRoutes/AllArticles.jsx

```diff
@@ -0,0 +1,37 @@
+import axios from "axios"
+import { useEffect, useState } from "react"
+
+export default function AllArticles() {
+    const [isLoading, setIsLoading] = useState(true)
+    const [articles, setArticles] = useState([])
+
+    useEffect(() => {
+        axios.get('https://nc-news-ee7w.onrender.com/api/articles?limit=100')
```

Comment:
I would consider extracting this into a seperate file and create an axios instance using axios.create, this is not essential but could save you a bunch of time in the future